tests/unit/restapi/v1/conftest.py

```diff
@@ -90,3 +90,34 @@ def registered_queues(
         "queue2": queue2_response,
         "queue3": queue3_response,
     }
+
+
+@pytest.fixture
+def registered_plugin_parameter_types(
+    client: FlaskClient, db: SQLAlchemy, auth_account: dict[str, Any]
+) -> dict[str, Any]:
+    # TODO: Move this to conftest.py after rebasing off dev
+    plugin_param_type1_response = actions.create_plugin_parameter_type(
+        client,
+        name="int",
+        group_id=auth_account["default_group_id"],
+        structure="",
+    )
```

Comment:
actions return `TestResponse` but the fixture should return a `dict`, so append `.get_json()` to each `create_plugin_parameter_type` call here.